logging/src/main/java/com/virtuslab/logger/PrefixedLambdaLogger.java

```diff
@@ -34,7 +39,9 @@ private String getLogMessagePrefix() {
     // #6: method from which IMacheteLogger.<logMethod> was called;
     @SuppressWarnings("upperbound")
     StackTraceElement element = Thread.currentThread().getStackTrace()[6];
-    return className + "#" + element.getMethodName() + ": ";
+    String timerMessage = bufferedTimerMessage.get();
```

Comment:
... and then `ThreadLocal#get` would be just sth like `.get(Thread.currentThread().getId())`

Reply:
Similarly with `set` and `remove`